src/libServer/LookupServer.cpp

```diff
@@ -2019,6 +2019,11 @@ Json::Value LookupServer::GetTransactionStatus(const string& txnhash) {
                              "Txn Hash size not appropriate");
     }
 
+    if (!REMOTESTORAGE_DB_ENABLE) {
```

Comment:
I suggest to make this the very first check, and also change the message to "API not supported" (remote storage is an internal detail)